### ISSUE TITLE
[Improvement] Applying blacklist to tags/keywords

### ISSUE THREAD

**Issue by Xalavar** [DragonOfMath/fa-blacklist#1]
I don't just mean the titles of artwork, by the way. It would seem feasible to make a GET request for each thumbnail link that populates on a webpage and leads back to the main page of the artwork submission (which it usually does), which follows the format:
`https://www.furaffinity.net/view/`

When it retrieves HTML data from that page, the script can then compare the keywords embedded in the HTML and check them against the user-specified blacklisted tags/filters. If there is a match, it should delete the image.

**Comment by Pentazer** [comment]
I agree, this would be great. The blacklist only works on the text in the post, but searching and blocking from the keywords would be better.
